Cesium.CodeGen/Ir/Expressions/SubscriptingExpression.cs

```diff
@@ -31,12 +31,14 @@ public IExpression Lower()
     public void EmitTo(IDeclarationScope scope) => Resolve(scope).EmitGetValue(scope);
 
     public IType GetExpressionType(IDeclarationScope scope) => Resolve(scope).GetValueType();
-
+    
```

Comment:
nit: can you remove this?

Reply:
To be honest, I don't care about the formatting _that_ much.